src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -14,29 +14,52 @@
 			@reset-filters="resetFilters"
 			@handle-overlay="handleQuickFiltersOverlay"
 		/>
+		<div
+			class="tw-flex tw-flex-wrap"
+		>
+			<kiva-classic-basic-loan-card
```

Comment:
this UI will also be cleaned up to match the figma... again, accidentally got ahead of myself 😂